### ISSUE TITLE
Adapty Flutter 3.2.0 (Android) Error: lateinit property channel has not been initialized

### ISSUE THREAD

**Issue by qwertyway** [adaptyteam/AdaptySDK-Flutter#121]
I'm having issue with the latest version 3.2.0 (Flutter) on android. I see this in logs during initialization 

```
E/GeneratedPluginRegistrant(15213): Error registering plugin adapty_flutter, com.adapty.flutter.AdaptyFlutterPlugin
E/GeneratedPluginRegistrant(15213): kotlin.UninitializedPropertyAccessException: lateinit property channel has not been initialized
E/GeneratedPluginRegistrant(15213): 	at com.adapty.flutter.AdaptyFlutterPlugin.onAttachedToEngine(AdaptyFlutterPlugin.kt:85)
E/GeneratedPluginRegistrant(15213): 	at com.adapty.flutter.AdaptyFlutterPlugin.onAttachedToEngine(AdaptyFlutterPlugin.kt:46)
E/GeneratedPluginRegistrant(15213): 	at io.flutter.embedding.engine.FlutterEngineConnectionRegistry.add(FlutterEngineConnectionRegistry.java:147)
E/GeneratedPluginRegistrant(15213): 	at io.flutter.plugins.GeneratedPluginRegistrant.registerWith(GeneratedPluginRegistrant.java:19)
E/GeneratedPluginRegistrant(15213): 	at java.lang.reflect.Method.invoke(Native Method)
E/GeneratedPluginRegistrant(15213): 	at io.flutter.embedding.engine.plugins.util.GeneratedPluginRegister.registerGeneratedPlugins(GeneratedPluginRegister.java:80)
E/GeneratedPluginRegistrant(15213): 	at io.flutter.embedding.engine.FlutterEngine.<init>(FlutterEngine.java:388)
E/GeneratedPluginRegistrant(15213): 	at io.flutter.embedding.engine.FlutterEngine.<init>(FlutterEngine.java:289)
E/GeneratedPluginRegistrant(15213): 	at io.flutter.embedding.engine.FlutterEngine.<init>(FlutterEngine.java:270)
E/GeneratedPluginRegistrant(15213): 	at io.flutter.embedding.engine.FlutterEngine.<init>(FlutterEngine.java:250)
E/GeneratedPluginRegistrant(15213): 	at io.flutter.embedding.engine.FlutterEngine.<init>(FlutterEngine.java:170)
E/GeneratedPluginRegistrant(15213): 	at io.flutter.plugins.firebase.messaging.FlutterFirebaseMessagingBackgroundExecutor.lambda$startBackgroundIsolate$0$io-flutter-plugins-firebase-messaging-FlutterFirebaseMessagingBackgroundExecutor(FlutterFirebaseMessagingBackgroundExecutor.java:139)
E/GeneratedPluginRegistrant(15213): 	at io.flutter.plugins.firebase.messaging.FlutterFirebaseMessagingBackgroundExecutor$$ExternalSyntheticLambda0.run(D8$$SyntheticClass:0)
E/GeneratedPluginRegistrant(15213): 	at android.os.Handler.handleCallback(Handler.java:958)
E/GeneratedPluginRegistrant(15213): 	at android.os.Handler.dispatchMessage(Handler.java:99)
E/GeneratedPluginRegistrant(15213): 	at android.os.Looper.loopOnce(Looper.java:205)
E/GeneratedPluginRegistrant(15213): 	at android.os.Looper.loop(Looper.java:294)
E/GeneratedPluginRegistrant(15213): 	at android.app.ActivityThread.main(ActivityThread.java:8177)
E/GeneratedPluginRegistrant(15213): 	at java.lang.reflect.Method.invoke(Native Method)
E/GeneratedPluginRegistrant(15213): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
E/GeneratedPluginRegistrant(15213): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:971)
```

**Comment by vladd-g** [comment]
Hi @qwertyway, thank you for the report!

Could you please check it on [this branch](https://github.com/adaptyteam/AdaptySDK-Flutter/tree/bugfix/android-channel-fix)?

**Comment by qwertyway** [comment]
@vladd-g the error is not showing on this branch, looks good

thank your for the quick reply and fix!

**Comment by vladd-g** [comment]
fixed in v3.2.1
